### PR TITLE
Fixes moment warning

### PR DESCRIPTION
**Pull request by kierangillen**
- Fixes moment deprecation warning on artwork page

### CODE REVIEW

File: src/Components/v2/AuctionCard.tsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ import { get } from "Utils/get"
 import { Media } from "Utils/Responsive"
 
 const zone = time => {
-  return moment(time).tz("America/New_York")
+  return moment(time, "YYYY-MM-DD").tz("America/New_York")
```
Comment (damassi) [inline src/Components/v2/AuctionCard.tsx]: 🚶

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 03d25d3fe3c4f9f03b602c4c185385b2794f6ca0

https://deploy-preview-1947--artsy-reaction.netlify.com

**Comment by damassi** [comment]
@kierangillen - can you fix the peril warning? Asignee is typically the spot for people who are designated to review / merge / approve, and Reviewers for people who you'd also like to have a look (but they're not obligated)

**Comment by kierangillen** [comment]
@damassi Woops that was a misclick, cool

**Pull request merged**